Cards/MadGraph5_aMCatNLO/DY/DYto2L-4Jets_MLL-10to50_madgraphMLM-pythia8/DYto2L-4Jets_MLL-10to50_madgraphMLM-pythia8_proc_card.dat

```diff
@@ -0,0 +1,14 @@
+import model sm-ckm_no_b_mass
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+
+generate p p > ell+ ell-  @0
+add process p p > ell+ ell- j  @1
+add process p p > ell+ ell- j j  @2
+add process p p > ell+ ell- j j j  @3
+add process p p > ell+ ell- j j j j  @4
```

Comment:
Double checking, we don't have H contributions from taus right?

Reply:
I tried but didn't spot any difference in diagrams.